mlflow/pyfunc/__init__.py

```diff
@@ -645,6 +648,16 @@ def __init__(
         else:
             self._predict_stream_fn = None
 
+    @property
+    @developer_stable
```

Comment:
Shall we mark langchain model `predict_with_callbacks` `predict_stream_with_callbacks`  devstable too ?

Reply:
I think we can hold off on this for now, as we may shift the implementation to a more generic pyfunc supported use case that can support callbacks as an optional param. 
When we do that, we'll just deprecate these existing patch-like methods in the langchain implementation. :)